spec/javascripts/collections/story_collection_spec.js

```diff
@@ -93,6 +93,27 @@ describe('StoryCollection', function() {
 
   });
 
+  describe("position on columns",function(){
```

Comment:
Why did you use double quotes here? Keep them just for cases where the description has the single quote on description itself and not just as a string wrapper. 💅

Reply:
More 💅  prefer to add spaces after commas also before curly braces for functions. Like:
`function() {`